src/simpla-admin.html

```diff
@@ -112,8 +135,16 @@
       }
 
       /**
+       * Attach singleton to window on created
+       * @return {[type]} [description]
+       */
+      ready() {
```

Comment:
Attached I think is just more robust. `ready()` might happen before or after `attached`, the timing is just a bit more confusing.